docs/website/docs/developers/general/release-management.md

```diff
@@ -63,10 +63,19 @@ request that some feature make the cut.
 
     * Set the tag to be created and select a target commit. For example, if the
         candidate release was tagged `iree-3.1.0rc20241119` at commit `3ed07da`,
-        set the new release tag `iree-3.1.0` and use the same commit.
+        set the new release tag `v3.1.0` and use the same commit.
```

Comment:
For the 3.0.0 release the tag was `iree-3.0.0` for the 3.1.0 release we have `v3.1.0`.

Reply:
That was intentional, since `vX.Y.Z` is more common across projects (left a note in #releasing on Discord).